tests/test_generate_itinerary.py

```
@@ -0,0 +1,138 @@
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)